automobile/scenario-creation-tutorial.md

```diff
@@ -78,6 +72,31 @@ Finally you need to write the [SUMO configuration file](http://sumo.dlr.de/wiki/
 </configuration>
 ```
 
+
 ### Add the SUMO interface
 
 Now that you have all the required files, you can open the generated world in Webots and add the `SumoInterface` PROTO (or a `Supervisor` node and associate the `sumo_supervisor` controller to it). Since the network files are already generated, you need to set the `useNetconvert` field to FALSE (or use the `--noNetconvert` option) and set in the `networkfiles` field (or use the `--d or --directory` option) the path to the directory where the SUMO network files are located.
```

Comment:
This text seems to come too early... We cannot yet open the generated world in Webots as it doesn't exist at this point... 😒